elephant/causality/granger.py

```diff
@@ -45,6 +45,7 @@
     :toctree: toctree/causality/
 
     pairwise_granger
+    conditional_granger
```

Comment:
add conditional_granger in `__all__` as well

Reply:
Addressed in commit https://github.com/NeuralEnsemble/elephant/pull/359/commits/bb4d8fe77adcbefa1bd4944c87376c4c1bedcbb5